src/lib/native.ts

```diff
@@ -757,3 +757,32 @@ export async function writePref(name: string, value: any) {
         write(file, text.replace(substr, `pref("${name}", ${value})`))
     }
 }
+
+/** Obey Mozilla's orders https://github.com/tridactyl/tridactyl/issues/1800 */
+export async function unfixamo() {
+    try {
+        if (localStorage.unfixedamo === "true") {
+            // fixamo already ran for the tridactyl instance in this profile
+            return;
+        }
+        const profile = (await getProfileDir()) + "/";
+        const userjs = await loadPrefs(profile + "user.js");
+        const tridactylPref = "tridactyl.unfixedamo"
+        if (userjs[tridactylPref] === "true") {
+            // fixamo already ran for this firefox profile
+            return;
+        }
+        const prefName = "privacy.resistFingerprinting.block_mozAddonManager"
+        if (userjs[prefName] !== undefined && userjs[prefName] !== "false") {
+            await writePref(prefName, "false");
```

Comment:
I think this is the wrong preference - blocking the add-on manager makes Firefox more secure, not less. It's the restricted domains they care about - see the RC https://github.com/tridactyl/tridactyl/commit/fef58f508f6da258ac2a447b02c86e33313b115b#diff-2eee6b5a0e6a1a2d81ecd725b12e2c8dR81-R84

Reply:
You're right that I should probably also fix the restricted domains list, however [this page](https://www.ghacks.net/2017/10/27/how-to-enable-firefox-webextensions-on-mozilla-websites/) disagrees with you about privacy.resistFingerprinting.block_mozAddonManager.

Reply:
Mozilla don't seem to care about it, presumably because it only affects old versions of Firefox.

Reply:
We __really__ shouldn't make people's browsers more identifiable if they have taken steps to make themselves more anonymous - I realise Tridactyl makes much of that moot but maybe those savvy users turn off Tridactyl when they want to be anonymous.

Restricted domains is a much less big deal.